src/mesh/generated/meshtastic/config.pb.h

```diff
@@ -214,7 +214,13 @@ typedef enum _meshtastic_Config_LoRaConfig_RegionCode {
     /* Malaysia 433mhz */
     meshtastic_Config_LoRaConfig_RegionCode_MY_433 = 16,
     /* Malaysia 919mhz */
-    meshtastic_Config_LoRaConfig_RegionCode_MY_919 = 17
+    meshtastic_Config_LoRaConfig_RegionCode_MY_919 = 17,
+    /* Singapore 433mhz */
+    meshtastic_Config_LoRaConfig_RegionCode_SG_433 = 18,
+    /* Singapore 868mhz */
+    meshtastic_Config_LoRaConfig_RegionCode_SG_868 = 19,
+    /* Singapore 923mhz */
+    meshtastic_Config_LoRaConfig_RegionCode_SG_923 = 20
```

Comment:
This change will be overwritten on next protocol buffers run, please submit an according PR to te protobuf repository

Reply:
Submitted PR to protobuf repository: [#430](https://github.com/meshtastic/protobufs/pull/430)